Code/src/main/java/nl/utwente/group10/ui/components/blocks/DisplayBlock.java

```diff
@@ -21,8 +28,8 @@
  * the value can be altered at any time by providing a different input source
  * using a Connection.
  */
-public class DisplayBlock extends Block {
-    /** The Block for which this DisplayBlock displays the output. **/
+public class DisplayBlock extends Block implements InputBlock, OutputBlock {
```

Comment:
In de functieblokconcepten van JJ en ondergetekende is een `DisplayBlock` zelf niet een `OutputBlock` (zie de sketches). Ik kan natuurlijk niet voor JJ spreken, maar ikzelf heb daarvoor gekozen zodat de bedoeling van een DisplayBlock meteen duidelijk is (alleen ankers aan de bovenkant). Van mij hoeft DisplayBlock dus niet OutputBlock te implementen.